devices/xiaomi.js

```diff
@@ -82,12 +82,46 @@ const fzLocal = {
                 case 0x0275:
                     result['valve_alarm'] = {1: true, 0: false}[value];
                     break;
+                case 0x00f7:
+                    result['valve_alarm'] ={1: true, 0: false}[value.readUInt32LE(36)];
+                    result['occupied_heating_setpoint'] = value.readUint16LE(32)/100;
+                    result['local_temperature'] = value.readUint16LE(28)/100;
+                    break;
+                case 0x027d:
+                    result['schedule'] = {1: 'ON', 0: 'OFF'}[value];
+                    break;
```

Comment:
How does this relate to the `preset` setting? Shouldn't the schedule be active when `preset` is set to `auto`, or does `auto` mean something different? What happens when I set `preset` to `manual` or `away`, and at the same time `schedule` to `ON`?